blocks/library/list/index.js

```diff
@@ -12,6 +12,40 @@ import Editable from '../../editable';
 
 const { children, prop } = hpq;
 
+function execCommand( command ) {
+	return ( { editor } ) => {
+		if ( editor ) {
+			editor.execCommand( command );
+		}
+	};
+}
+
+function listIsActive( listType ) {
+	return ( { nodeName = 'OL', internalListType } ) => {
+		return listType === ( internalListType ? internalListType : nodeName );
+	};
+}
+
+function listSetType( listType, editorCommand ) {
+	return ( { internalListType, editor }, setAttributes ) => {
+		if ( internalListType ) {
+			// only change list types, don't toggle off internal lists
+			if ( internalListType !== listType )	{
+				if ( editor ) {
+					editor.execCommand( editorCommand );
+				}
+			}
+		} else {
+			setAttributes( { nodeName: listType } );
+		}
+	};
+}
+
+function findInternalListType( { parents } ) {
+	const list = parents.find( ( node ) => node.nodeName === 'UL' || node.nodeName === 'OL' );
```

Comment:
We don't currently use a polyfill for prototype methods on base object types, so `Array#find`, `String#includes` et. all will fail in Internet Explorer 11. I think we _will_ want a polyfill, mostly because it's confusing for developers to have access to some but not all ES2015+ features. I'd started to explore this in #746, but closed it in anticipation of an upcoming `babel-preset-env` "usage" feature.

...which is all to say, this will probably be fine, but cause Internet Explorer to fail for some time. Alternative is to use `import { find } from 'lodash';`.